sdk/tables/azure-data-tables/setup.py

```diff
@@ -67,18 +74,17 @@
     ],
     zip_safe=False,
     packages=find_packages(exclude=[
+        'tests',
         # Exclude packages that will be covered by PEP420 or nspkg
         'azure',
-        'tests',
+        'azure.data',
     ]),
     install_requires=[
-        "azure-core<2.0.0,>=1.2.2",
-        "msrest>=0.6.10"
-        # azure-data-tables
+        'msrest>=0.5.0',
```

Comment:
assuming lower dependency should stay the same. You should also still need azure-core dependency